src/color-chart/color-chart.js

```diff
@@ -478,7 +507,7 @@ const Self = class ColorChart extends ColorElement {
 					let range = this.xResolved?.refRange ?? this.xResolved?.range ?? [0, 100];
 					return range[0];
 				}
-				else if (!this.x || this.xMin === "auto") {
+				else if ((!this.x && !this.force.x.min) || this.xMin === "auto") {
```

Comment:
While it's not _particularly_ slow, let's not be wasteful; I'd assign `this.force` to a variable rather than invoking the getter every time

Reply:
@LeaVerou, do you mean like so? Does it give us any advantages if we do that in prop getters?

<img width="641" height="259" alt="image" src="https://github.com/user-attachments/assets/04563696-6faf-449d-98b0-4c2005246f80" />